pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -135,20 +197,14 @@ rec {
     };
   };
 
-  antlr-runtime_4_5 = buildEclipsePluginBase rec {
+  antlr-runtime_4_5 = buildEclipsePlugin rec {
     name = "antlr-runtime-4.5.3";
 
-    src = fetchurl {
+    srcPlugin = fetchurl {
       url = "https://www.antlr.org/download/${name}.jar";
       sha256 = "0lm78i2annlczlc2cg5xvby0g1dyl0sh1y5xc2pymjlmr67a1g4k";
     };
 
-    buildCommand = ''
-      dropinDir="$out/eclipse/dropins/"
-      mkdir -p $dropinDir
-      cp -v $src $dropinDir/${name}.jar
-    '';
-
     meta = with lib; {
       description = "A powerful parser generator for processing structured text or binary files";
```

Comment:
Not sure why these were using buildEclipsePluginBase to begin with, buildEclipsePlugin would always have been simpler. But this is now a forced move, since buildEclipsePluginBase doesn't work any more.